### PR TITLE
modify datanorm op test=develop

### PR DESCRIPTION
**Pull request by yaoxuefeng6**
Add enable_scale_and_shift option in datanorm op
By setting enable_scale_and_shift, this op will do scale and shift after normalization like batchnorm.

### CODE REVIEW

**Review by xjqbest — DISMISSED** [review]
good

**Review by saxon-zh — DISMISSED** [review]

File: paddle/fluid/framework/unused_var_check.cc [hunk 1]
```diff
@@ -53,7 +53,9 @@ const std::unordered_set<std::string> op_has_unsed_vars_white_list = {
     "precision_recall",                // 1
     "fusion_seqpool_cvm_concat",       // 2
     "fused_batch_norm_act",            // 2
-    "fused_batch_norm_act_grad"        // 2
+    "fused_batch_norm_act_grad",       // 2
+    "data_norm",
```
Comment (zhiqiu) [inline paddle/fluid/framework/unused_var_check.cc]: Please add comments to specify the type, like other op.

**Review by zhiqiu — APPROVED** [review]
LGTM for unused_var_check.cc

File: paddle/fluid/framework/unused_var_check.cc (hunk 1 above)
Comment (yaoxuefeng6) [reply paddle/fluid/framework/unused_var_check.cc]: added

**Review by fuyinno4 — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by saxon-zh — APPROVED** [review]

**Pull request merged**